scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  color: $black;
+  font-size: $title-size;
+  height: 220px;
+  margin-left: 80px;
+
+  &::before{
+    content: url(assets/triangle.svg);
+    display: block;
+    position: relative;
```

Comment:
Have you tried using absolute position on this element? I think that it would be better to use this position when what you're positioning is something decorative. 🙂